### PR TITLE
Phalanxguard patch 1

### PR DESCRIPTION
**Pull request by Phalanxguard**

### CODE REVIEW

File: pytrading212/trading212.py [hunk 1]
```diff
@@ -57,10 +57,17 @@ def __init__(
         self.driver = driver
 
         # authenticate
+        
         self.driver.get("https://www.trading212.com/en/login")
         self.driver.find_element_by_name("email").send_keys(username)
         self.driver.find_element_by_name("password").send_keys(password)
-        self.driver.find_element_by_class_name("submit-button").click()
+        time.sleep(1) #Let the Accept all cookies popup arrive
```
Comment (HellAmbro) [inline pytrading212/trading212.py:64]: I think this sleep can be safely deleted.

File: pytrading212/trading212.py [hunk 2]
```diff
@@ -12,16 +12,16 @@
...
 from selenium.webdriver.support.ui import WebDriverWait
 
 from pytrading212.order import ValueOrder
 from pytrading212.position import Position
 
 
 class Mode(Enum):
-    DEMO = "demo"
-    LIVE = "live"
+    DEMO = ("demo",)
```
Comment (HellAmbro) [inline pytrading212/trading212.py:23]: Why are you using round brackets?

**Review by HellAmbro — COMMENTED** [review]
Thanks for fixing cookie popup.

File: pytrading212/trading212.py (hunk 2 above)
Comment (Phalanxguard) [reply pytrading212/trading212.py:23]: No idea, I think those must have been in the original code, I didn't intentionally add it!

Comment (HellAmbro) [reply pytrading212/trading212.py:23]: Yes, maybe you're using an older version.

**Pull request merged**